frontend/package.json

```diff
@@ -90,16 +90,17 @@
 		"dev:reflame:svgr": "node scripts/build-svgr.mjs --watch",
 		"dev:tsm": "tsm src -w --logLevel=error",
 		"dev:gql": "graphql-codegen --config --watch codegen.yml",
-		"reflame": "node scripts/update-reflame-environment.mjs && run-p --print-label --race 'dev:reflame:*'",
 		"codegen": "graphql-codegen --config codegen.yml",
-		"build": "node scripts/update-reflame-environment.mjs && run-p --print-label 'build:**'",
+		"build": "yarn reflame-env && run-p --print-label 'build:**'",
 		"build:vite": "yarn types:check && NODE_OPTIONS='--max-old-space-size=7168' vite build",
 		"build:reflame:css": "node scripts/build-css-bundle.mjs",
 		"build:reflame:scss": "node scripts/build-scss-modules.mjs",
 		"build:reflame:ve": "node scripts/build-vanilla-extract.mjs",
 		"build:reflame:rrweb": "node scripts/build-rrweb.mjs",
 		"build:reflame:svgr": "node scripts/build-svgr.mjs",
-		"reflame-build": "node scripts/update-reflame-environment.mjs && run-p --print-label 'build:reflame:*'",
+		"reflame-env": "which doppler && (doppler run -c prd --command='node scripts/update-reflame-environment.mjs production' && doppler run -c docker --command='node scripts/update-reflame-environment.mjs docker') || exit 0",
```

Comment:
this should be `prod_aws_render` for prod and `dev` for dev

Reply:
Updated here: https://github.com/highlight/highlight/pull/5329/commits/09647438441f7cb4fa23e441d8c029383397fd73

Mind pulling this down, running `yarn reflame-build` and committing the result to test this out when you have the chance? Could also merge this and do that in a separate PR if you prefer.

Reply:
done!